_operator_reference/providers.md

```diff
@@ -822,35 +824,36 @@ Kubernetes:
-- `kinds`: # (V2 Only) A list of resource kinds this Spinnaker account can deploy to and will cache. When no kinds are configured, this defaults to 'all kinds described here https://spinnaker.io/reference/providers/kubernetes-v2/'.
-- `omitKinds`: # (V2 Only) A list of resource kinds this Spinnaker account cannot deploy to or cache. This can only be set when --kinds is empty or not set.
-- `customResources`: # (V2 Only) List of Kubernetes custom resources to managed by clouddriver and made available for use in patch and delete manifest stages.
-- `kubernetesKind`: abc # Fully qualified name of the Kubernetes CRD
-- `spinnakerKind`: instances # One of instances, configs, serverGroups, loadBalancers, securityGroups, serverGroupManagers, unclassified
-- `versioned`: false
+- `context`: The kubernetes context to be managed by Spinnaker. See http://kubernetes.io/docs/user-guide/kubeconfig-file/#context for more information. When no context is configured for an account the 'current-context' in your kubeconfig is assumed.
+- `cluster`: Used with V1 provider (deprecated)
+- `user`: Used with V1 provider (deprecated)
+- `configureImagePullSecrets`: Used with V1 provider. When true, Spinnaker will create & manage your image pull secrets for you; when false, you will have to create and attach them to your pod specs by hand.
+- `serviceAccount`: When true, Spinnaker attempt to authenticate against Kubernetes using a Kubernetes service account. This only works when Halyard & Spinnaker are deployed in Kubernetes. Read more about service accounts here: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/.
+- `cacheThreads`: Number of caching agents for this kubernetes account. Each agent handles a subset of the namespaces available to this account. By default, only 1 agent caches all kinds for all namespaces in the account.
+- `namespaces`: A list of namespaces this Spinnaker account can deploy to and will cache. When no namespaces are configured, this defaults to 'all namespaces'.
+- `omitNamespaces`: A list of namespaces this Spinnaker account cannot deploy to or cache. This can only be set when --namespaces is empty or not set.
+- `kinds`: (V2 Only) A list of resource kinds this Spinnaker account can deploy to and will cache. When no kinds are configured, this defaults to 'all kinds described here https://spinnaker.io/reference/providers/kubernetes-v2/'.
+- `omitKinds`: (V2 Only) A list of resource kinds this Spinnaker account cannot deploy to or cache. This can only be set when --kinds is empty or not set.
+- `customResources`: (V2 Only) List of Kubernetes custom resources to managed by clouddriver and made available for use in patch and delete manifest stages.
+	- `kubernetesKind`: Fully qualified name of the Kubernetes CRD
+	  - `versioned`: true or false
+	- `spinnakerKind`: One of instances, configs, serverGroups, loadBalancers, securityGroups, serverGroupManagers, unclassified
+	  - `versioned`: true or false
```

Comment:
`versioned` is only a child of `customResources`:
```
            customResources: # (V2 Only) List of Kubernetes custom resources to managed by clouddriver and made available for use in patch and delete manifest stages.
            - kubernetesKind: abc # Fully qualified name of the Kubernetes CRD
              spinnakerKind: instances # One of instances, configs, serverGroups, loadBalancers, securityGroups, serverGroupManagers, unclassified
              versioned: false
```

Reply:
I think these are correct now. I replaces tabs with spaces, and that screwed up the indentation in some places.